### PR TITLE
Show clipboard contents while typing

### LINKED ISSUE #1118: Show clipboard URL as user starts typing
_Added by @bbinto_

**Why**
- Provide users with a convenient way to quickly access the URL they just copied to the clipboard earlier

**What**
-  We detect that the string in the clipboard is a URL.
-  If true, then show “Link you copied: [url.com]” as soon as you type something into the URL bar.
- Tap on the URL shown to go to the URL.

**Acceptance Criteria (how do I know when I’m done?)**
- Basic flow as described above can be performed
- Probes included to show how often users 
   - come into the app, having a URL copied to their clipboard
   - use the feature, i.e. Tap on the URL

--
We have this in iOS right now and it seems to be a great way to provide value given the context of Focus. It's also a quick way to help users move links into Focus. But we're still learning and iterating on this feature. 

Filing this to see if we also want to bring this over to Android. @bbinto thoughts?

Latest bug here for iOS: https://github.com/mozilla-mobile/focus-ios/issues/269

### PR DESCRIPTION
**Pull request by jonalmeida**
Fixes #1118 

I wrote this without seeing #1184, so close this if won't be used.  ¯\_(ツ)_/¯

### CODE REVIEW

File: app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java [hunk 1]
```diff
@@ -45,7 +46,7 @@
 /**
  * Fragment for displaying he URL input controls.
  */
-public class UrlInputFragment extends LocaleAwareFragment implements View.OnClickListener, InlineAutocompleteEditText.OnCommitListener, InlineAutocompleteEditText.OnFilterListener, PopupMenu.OnMenuItemClickListener {
+public class UrlInputFragment extends LocaleAwareFragment implements View.OnClickListener, InlineAutocompleteEditText.OnCommitListener, InlineAutocompleteEditText.OnFilterListener, PopupMenu.OnMenuItemClickListener, ClipboardManager.OnPrimaryClipChangedListener {
```
Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[PMD\] INFO: The class 'UrlInputFragment' has a Cyclomatic Complexity of 3 \(Highest = 25\)\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[PMD\] INFO: The class 'UrlInputFragment' has a Modified Cyclomatic Complexity of 3 \(Highest = 25\)\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[PMD\] INFO: The class 'UrlInputFragment' has a Standard Cyclomatic Complexity of 3 \(Highest = 25\)\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[PMD\] INFO: This class has too many methods, consider refactoring it\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java]: \[PMD\] INFO: Too many fields

**Comment by pocmo** [comment]
This whole feature is on hold. The iOS version just pulled it.

**Pull request closed without merging**